src/OrchardCore.Modules/OrchardCore.Templates/Views/Template/Edit.cshtml

```diff
@@ -40,7 +40,16 @@
 
     <fieldset>
         <div class="form-group">
-            <button class="btn btn-primary" type="submit">@T["Save"]</button>
+            <div class="btn-group">
+                <button type="submit" name="submit" class="btn btn-secondary">@T["Save"]</button>
+                <button type="button" class="btn btn-secondary dropdown-toggle dropdown-toggle-split" data-reference="parent" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <span class="sr-only">@T["Toggle Dropdown"]</span>
```

Comment:
What is this line, and why is it translated?

Reply:
Should we finally get rid of this translation?
I made a seach and we only used this "sr-only" class a couple of times. One of them in, the Mvc.Web project,  is not translated.  And the other one, in theBlogTheme, we do translate it.